### PR TITLE
Updated readmes and contribution document for easier engineer onboarding, added IPFS secret

### PR DESCRIPTION
**Pull request by paul-bdio**

### CODE REVIEW

**Comment by SkandaBhat** [comment]
Not sure if it was an isolated problem, but I had issues installing `fastecdsa`. This SO helped me resolve it - https://stackoverflow.com/a/70607294.

Maybe there's merit to adding this in the readme or maybe under "Troubleshooting"?

I am running an M1 Macbook Air.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Setting up the repository
+
+1. Clone the repository.
+
+```jsx
+git clone https://github.com/zorro-project/zorro.git
+```
+
+1. Run “yarn install” in /starknet and /redwood/web
```
[the remark this answers is not in the document]
Comment (valeseus) [reply CONTRIBUTING.md:29]: doesn't /redwood have a different package.json than /redwood/web? or does redwood handle that for you?

**Comment by valeseus** [comment]
Sorry guys, didn't realize I was on the wrong GitHub account with this PR. I've opened a new PR here - https://github.com/zorro-project/zorro/pull/54, resolved those issues and added the troubleshooting part like @SkandaBhat mentioned. @corbt, Only question I have is on my comment about the "yarn install". And I'm leaving the JSX tag for easier copy/paste.

**Comment by corbt** [comment]
Closing in favor of https://github.com/zorro-project/zorro/pull/54 and https://github.com/zorro-project/zorro/pull/53

**Pull request closed without merging**